.travis.yml

```diff
@@ -13,8 +13,6 @@ before_install:
 install:
     - pip3 install flake8 -r requirements.txt
     - flake8 . --select=E9,F63,F7,F82 --show-source --statistics
-    - sudo env "PATH=$PATH" make clean
```

Comment:
`make clean` is used to clean up prior installations. Should we keep it? :thinking:

Reply:
What needs to be cleaned up?

The udev rules possibly, in case they are updated. I'll check if there is a way to get pip to remove them on uninstall/upgrade.

Anything else?

Reply:
Actually, does `make clean` even remove the udev rules? It looks like it just removes the python package. If so, `pip uninstall PSL` will accomplish the same thing.

Reply:
For Travis, specifically, there is no need to clean up anything since it [does not preserve state](https://docs.travis-ci.com/user/common-build-problems/#travis-ci-does-not-preserve-the-state-between-builds) between builds.

Reply:
Actually I was referring to the Makefile. Should we keep it?

Reply:
Unless the Makefile prodives additional functionality beyond what pip offers (which I don't think it does), we shouldn't keep it.

Reply:
That makes sense :+1: